pkg/controller/proposal/controller.go

```diff
@@ -818,3 +820,24 @@ func isModelDataCompatible(pluginDataModels []*gpb.ModelData, targetDataModels [
 	}
 	return true
 }
+
+func doCascadingDelete(changeValues map[string]*configapi.PathValue, configStore map[string]*configapi.PathValue) map[string]*configapi.PathValue {
```

Comment:
I think a better name might be `addDeletedChildren` and the comment should go on the line before

Reply:
Renamed